target/linux/x86/generic/config-5.4

```diff
@@ -139,6 +139,7 @@ CONFIG_FB_DEFERRED_IO=y
 CONFIG_FB_EFI=y
 CONFIG_FB_HYPERV=y
 # CONFIG_FB_I810 is not set
+CONFIG_FB_SIMPLE=y
```

Comment:
How is this related to this?

Reply:
It is related to uefi firmware display driver.Without this , the console has no output.

Reply:
It is related to uefi firmware display driver.Without this , the console has no output.

Reply:
https://git.openwrt.org/?p=openwrt/openwrt.git;a=commitdiff;h=2712497442cc322aa9a6e166a7ed850a4d42df71

Reply:
Ok, thanks for info